app/assets/stylesheets/ama_layout/layout_components/forms.scss

```diff
@@ -224,11 +218,24 @@ input[type="checkbox"]:checked + label{
   float: right;
 }
 
-p.hint{
-  font-size: $base-font-size*0.75;
-  line-height: $base-font-size;
-  color: darken($smoke, 15%);
-  font-style: italic;
+/*-------------- Credit Card images ---------- */
+.cc-type{
+  width: 38px;
+  height: 19px;
+  display: inline-block;
+}
+
+.type-visa{
+  background: url(image-path("cc-type-visa.png")) no-repeat;
+  background-size: contain;
+  background-position: center center;
+}
+
+.type-mc,
+.type-mastercard{
```

Comment:
Why do we need both "mastercard" and "mc"? Are different apps using different class names?